docs/source/create_channel/create_channel_test_net.md

```diff
@@ -0,0 +1,444 @@
+- **`--client-key`**: Specify the location and file name of admin client private key from the TLS CA.
+
+When successful, the output of the command contains the following:
+```
+Status: 201
+{
+	"name": "channel1",
+	"url": "/participation/v1/channels/channel1",
+	"consensusRelation": "consenter",
+	"status": "active",
+	"height": 1
+}
+```
+
+The channel is active and ready for peers to join.
+
+### Consenter vs. Follower
+
+Notice the ordering node was joined to the channel with a `consensusRelation: "consenter"`. If you ran the command against an ordering node that is not included in the list of `Consenters:` in the `configtx.yaml` file (or the channel configuration consenter set), it is added to the channel as a `follower`. To learn more about considerations when joining additional ordering nodes see the topic on [Joining additional ordering nodes](create_channel_participation.html#step-three-join-additional-ordering-nodes).
+
+### Active vs. onboarding
+
+An orderer can join the channel by providing the channel **genesis block**, or the **latest config block**. If joining from the latest config block, the orderer status is set to `onboarding` until the channel ledger has caught up to the specified config block, when it becomes `active`. At this point, you could then add the orderer to the channel consenter set by submitting a channel update transaction, which will cause the  `consensusRelation` to change from `follower` to `consenter`.
+
+## Next steps
+
+After you have created the channel, the next steps are to join peers to the channel and deploy smart contracts. This section walks you through those processes using the test network.
+
+### List channels on an orderer
+
+Before you join peers to the channel, you might want to try to create additional channels. As you create more channels, the `osnadmin channel list` command is useful to view the channels that this orderer is a member of. The same parameters are used here as in the `osnadmin channel join` command from the previous step:
+```
+osnadmin channel list -o localhost:7053 --ca-file "$ORDERER_CA" --client-cert "$ORDERER_ADMIN_TLS_SIGN_CERT" --client-key "$ORDERER_ADMIN_TLS_PRIVATE_KEY"
+```
+The output of this command looks similar to:
+
+```
+Status: 200
+{
+	"systemChannel": null,
+	"channels": [
+		{
+			"name": "channel1",
+			"url": "/participation/v1/channels/channel1"
+		}
+	]
+}
+```
+
+### Join peers to the channel
+
+The test network includes two peer organizations each with one peer. But before we can use the peer CLI, we need to set some environment variables to specify which user (client MSP) we are acting as and which peer we are targeting. Set the following environment variables to indicate that we are acting as the Org1 admin and targeting the Org1 peer.
+
+```
+export CORE_PEER_TLS_ENABLED=true
+export PEER0_ORG1_CA=${PWD}/organizations/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
```

Comment:
No need to set env variable PEER0_ORG1_CA.
Just set the path in CORE_PEER_TLS_ROOTCERT_FILE directly.
Unnecessary indirection only adds confusion.